staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -236,6 +236,15 @@ type Interface interface {
 
 	// Count returns number of different entries under the key (generally being path prefix).
 	Count(key string) (int64, error)
+
+	// RequestBookmark requests etcd to send a bookmark to be sent on selected watches.
+	// In response etcd should send a bookmark informing the current progress on watch.
+	// Only watches with matching context grpc metadata (https://github.com/kubernetes/kubernetes/blob/9325a57125e8502941d1b0c7379c4bb80a678d5c/vendor/go.etcd.io/etcd/client/v3/watch.go#L1037-L1042)
+	// will be notified. Useful for resources with unfrequented changes with long periods of no events received.
```

Comment:
Please update the comment to describe the semantic (the question I was asking - what if watch is lagging, etc.)

Reply:
ping

Reply:
Sorry, was interrupted before sending the update.